sherpa/plot/tests/test_pylab_specific.py

```diff
@@ -0,0 +1,59 @@
+#
+#  Copyright (C) 2022, 2023
+#  MIT
+#
+#
+#  This program is free software; you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation; either version 3 of the License, or
+#  (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License along
+#  with this program; if not, write to the Free Software Foundation, Inc.,
+#  51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+'''This module collects tests specific to matplotlib.
+
+The purpose here is not to throw all tests that work with matplotlib into
+a single file, but instead to provide a space for tests that do not so much
+test sherpa plotting functionality, but that just look at specific
+implementation details in pylab_backend.py.
+
+That is not always a clear line, so as a rule of thumb, existing tests
+remain in their old place, only newly written tests will be placed here.
+'''
+
+import numpy as np
+import pytest
+from sherpa import data
+from sherpa import plot
+
+
+def test_PylabErrorArea():
```

Comment:
My original concern has been cleared up, but we can add a check to ensure we don't get any messages from the `TemporaryPlottingBakend` call with the following changes (1 of 2) 

```suggestion
def test_PylabErrorArea(caplog):
```